src/Entity/EntityBase.php

```diff
@@ -147,10 +147,7 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
       ->setDescription(t('The entity type.'))
       ->setSetting('target_type', $entity_type->getBundleEntityType());
 
-    $fields['revision_log'] = BaseFieldDefinition::create('string_long')
-      ->setLabel(t('Revision log message'))
-      ->setDescription(t('The log entry explaining the changes in this revision.'))
-      ->setRevisionable(TRUE);
+    $fields += static::entityRevisionLogBaseFieldDefinitions();
```

Comment:
Ah I see why you are confused. You probably used the drupal core 8.1.x ... which adds those fields automatically now, see the other PR. This PR is just about fixing some failures + getting it working with the last 8.0.x/8.x-0.x version